Libraries/Text/Text/RCTTextView.m

```diff
@@ -173,6 +184,13 @@ - (void)setTextStorage:(NSTextStorage *)textStorage
 - (void)drawRect:(CGRect)rect
 {
   [super drawRect:rect];
+
+  // For iOS, UITextView api is not used for legacy performance reasons. A custom draw implementation is used instead.
+  // On dekstop, we use NSTextView to access api's for arbitrary selection, custom cursors etc...
```

Comment:
typo `desktop`

Reply:
fixed in b72d4bcaf5e482feeb784494cb2ab069a0aa26fe